packages/cozy-sharing/src/SharingProvider.jsx

```diff
@@ -217,12 +217,15 @@ export class SharingProvider extends Component {
         readOnlyRecipients
       })
 
+    const previewPath =
+      document._type === 'com.bitwarden.organizations' ? undefined : '/preview'
+
```

Comment:
I think I already commented something different but here is my new mind 🙄 : 

previewPath should be set by the caller. Like that, we can do what we want from the app's land. 

```jsx
 <SharingProvider doctype="io.cozy.files" documentType="Files" previewPath="/preview">
```

Reply:
Seems doable. Maybe we should set a default to "/preview" to maintain retro-compatiblity?

Reply:
I think so, yes!

Reply:
Implemented in https://github.com/cozy/cozy-libs/pull/1326/commits/66bd98e8f26aeac6c645ea604c13bf2ccc91aeaf

I never unit-tested React Providers before, do you know an easy way to test this part?